homeassistant/components/mqtt/server.py

```diff
@@ -85,6 +98,6 @@ def generate_config(hass, passwd):
         username = None
         password = None
 
-    client_config = ('localhost', 1883, username, password, None, PROTOCOL_311)
+    client_config = ('localhost', 1883, username, password, None, '3.1.1')
```

Comment:
I moved the HBMQTT schema to this file (it makes a lot more sense to have it with the rest of the HBMQTT server specific code). So `_init_.py` now imports from `server.py`

To prevent circular imports (`PROTOCOL` lives in `_init_.py`) I removed this constant (but must say I haven't tested if circular imports is bad :-) just remembered something from the HASS Core refactoring that it shouldn't be done)